ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-commands.sh

```diff
@@ -4,19 +4,14 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-function checkCli() {
-    export IBMCLOUD_CLI=ibmcloud
-    export IBMCLOUD_HOME=/output
-    echo "check IBMCLOUD_CLI: ${IBMCLOUD_CLI}..."
-    command -v ${IBMCLOUD_CLI}
-    ${IBMCLOUD_CLI} --version
-    ${IBMCLOUD_CLI} plugin list
-}
-
 # IBM Cloud CLI login
 function ibmcloud_login {
```

Comment:
I suggest not to place the lines in a function which is only called one time in this step, but directly run the lines. And in prow ci, we already set -o errexit, a function may skip the error happen in the function.